### PR TITLE
Remove unused method parameter

### PR DESCRIPTION
**Pull request by ssoloff**

### CODE REVIEW

**Comment by ssoloff** [comment]
@ron-murhammer So here's the history on the `forceInMemory` parameter...

* Usage removed in #3347.  Would be first occurrence of the unused parameter warning.
* Entire `updateUndrawnTiles()` method eliminated in #3482 (this PR was the starting point for the tile drawing optimization revert).
* Tile drawing optimization revert in #3725.  This re-introduced the `updateUndrawnTiles()` method in the state it was in prior to #3482 with the unused parameter.

**TL;DR:** The functionality that made use of that parameter was removed in #3347, which was not part of the revert in #3725.  I don't know if that functionality is something we want to restore, but it was not apparently at the root of the drawing problems we reverted.
